pkg/assets/assets_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assets_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/kinvolk/lokomotive/pkg/assets"
+)
+
+// Verify assets are successfully extracted from memory to disk.
+func TestExtract(t *testing.T) {
+	dir, err := ioutil.TempDir("", "lokoctl")
+	if err != nil {
+		t.Fatalf("Creating temp dir: %v", err)
+	}
+
+	t.Cleanup(func() {
+		if err := os.RemoveAll(dir); err != nil {
+			t.Logf("Could not remove temp dir %q", dir)
```

Comment:
It's suspicious if that ever happens, so would be good to log why it failed.
```suggestion
			t.Logf("Could not remove temp dir %q: %v", dir, err)
```

Reply:
This was an oversight. Fixed.